content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
 -------------------------------------
 
-Once the product is in the :guilabel:`Input Location`, the internal transfer is ready to move the
-product to :guilabel:`Quality Control`. In the :menuselection:`Inventory` :guilabel:`Overview`
-dashboard, click the :guilabel:`1 To Process` smart button in the :guilabel:`Internal Transfers`
-kanban card.
+Once the receipt has been validated, an internal transfer operation to move the product to quality
+control is ready to process.
 
-.. image:: receipts_three_steps/three-step-quality-transfer.png
-   :align: center
-   :alt: One Internal Transfer ready to process in the Inventory Overview kanban view.
+Click :guilabel:`Inventory Overview` in the breadcrumbs to navigate back to the dashboard, and
+locate the :guilabel:`Internal Transfers` task card.
+
+Click the :guilabel:`# To Process` button to reveal all internal transfers to process. Then, click
+the internal transfer associated with the validated receipt.
 
-Click on the :guilabel:`Transfer` associated with the purchase order, then click
-:guilabel:`Validate` to complete the transfer and move the product to the :guilabel:`Quality
-Control` location. Once the transfer is validated, the product is ready for the quality inspection,
-but is not available for manufacturing or delivery orders.
+Click :guilabel:`Validate` to complete the transfer, and move the product from :guilabel:`WH/Input`
+to :guilabel:`WH/Quality Control`.
 
-.. image:: receipts_three_steps/validate-three-step-quality-move.png
+.. image:: receipts_three_steps/receipts-three-steps-internal-transfer.png
    :align: center
-   :alt: Validate the internal transfer to move the item to the Quality Control location.
+   :alt: Internal transfer for product being moved to quality control zone.
 
 Process a transfer to stock
 ===========================
 
-Once the product is in the :guilabel:`Quality Control` location, the final internal transfer is
-ready to move the product to :guilabel:`Stock`. In the :guilabel:`Inventory` overview dashboard,
-click the :guilabel:`1 To Process` smart button in the :guilabel:`Internal Transfers` Kanban card.
+Once the internal transfer to move the product to quality control has been validated, another
+internal transfer operation to move the product into warehouse stock is ready to process.
+
+Click :guilabel:`YourCompany: Internal Transfers` in the breadcrumbs to reveal the list of all
+internal transfers to process. Then, click the new internal transfer to move the product from
+`WH/Quality Control` to `WH/Stock`.
 
-Click on the final :guilabel:`Transfer` associated with the purchase order, then click
-:guilabel:`Validate` to complete the transfer and move the product to stock. Once the transfer is
-validated, the product enters the stock and is available for customer deliveries or manufacturing
-orders.
+Click :guilabel:`Validate` to complete the transfer, and move the product from :guilabel:`WH/Quality
+Control` to :guilabel:`WH/Stock`.
```

Comment:
Same here: **Click**.

Reply:
Changed some words in the second and third paragraphs per the instructions from this suggestion 🙂 Suggestion implemented ✅